perun-openapi/openapi.yml

```
@@ -2329,6 +2329,87 @@ components:
       in: query
       required: true
 
+    namespace:
+      name: namespace
+      description: "namespace"
+      schema:
+        type: string
+      in: query
+      required: true
+
+    firstName:
+      name: firstName
+      description: firstName of user
+      schema:
+        type: string
+      in: query
+      required: true
+
+    lastName:
+      name: lastName
+      description: lastName of user
+      schema:
+        type: string
+      in: query
+      required: true
+
+    titleBefore:
+      name: titleBefore
+      description: title before name of user
+      schema:
+        type: string
+      in: query
+      required: false
+
+    titleAfter:
+      name: titleAfter
+      description: title after name of user
+      schema:
+        type: string
+      in: query
+      required: false
+
+    password:
+      name: password
+      description: password
+      schema:
+        type: string
+      in: query
+      required: true
+
+    sponsor:
```

```suggestion
    sponsorId:
```
I would change the name so it is consistent with other id parameters.